mod_reforged/hooks/items/armor_upgrades/bone_platings_upgrade.nut

```diff
@@ -0,0 +1,7 @@
+::Reforged.HooksMod.hook("scripts/items/armor_upgrades/bone_platings_upgrade", function(q) {
+	q.onCombatStarted = @() function()
+	{
+		this.m.IsUsed = true;	// (vanilla) Disable the built-in damage reduction from the item
```

Comment:
What if a submod wants to add an additional effect to this item and checks for `this.m.IsUsed`? In order to have better submod compatibility with regards to submod compatibility perhaps a better way to do it would be leave `this.m.IsUsed` untouched and instead overwrite the vanilla `onBeforeDamageReceived` function to be empty:
```squirrel
// Overwrite the vanilla function to be empty so that damage reduction isn't applied from this item
q.onBeforeDamageReceived = @() function( _attacker, _skill, _hitInfo, _properties )
{
}
```